route/config/route_config_params.yaml

```diff
@@ -4,4 +4,7 @@ max_crosstrack_error: 25
 
```

Comment:
please see line 3. This probably now has to change now that it is not measuring form the center of the road but from bounding lines. Please confirm with Mike or Kyle

Reply:
Agreed, this comment should be updated to reflect that, and the value should probably be trimmed way down to something reasonable like 2-4m, I'd think.